blackjax/smc/particle_utils.py

```diff
@@ -0,0 +1,23 @@
+"""
```

Comment:
`*_utils.py` is a design smell. In this case it's totally fine to duplicate `num_particles`.